blocks/header/header.css

```diff
@@ -1434,3 +1434,331 @@ header .l3-nav-desktop .right-sec-menu-option:last-child {
+  text-align: right
+}
+
+.ecommerce-header-cta-exceeded,.ecommerce-header-cta-order-details {
+  margin-left: 2rem
+}
+
+.ecommerce-header-logo {
+  -webkit-box-pack: start;
+  -ms-flex-pack: start;
+  justify-content: flex-start
+}
+
+.ecommerce-header .vlt-site-logo {
+  color: #fff;
+  display: flex;
+  height: 2.8rem;
+  margin-right: 2rem
+}
+
+.ecommerce-header .vlt-site-logo.partner-desktop {
+  display: none
+}
+
+.ecommerce-header .vlt-site-logo.partner-mobile {
+  display: flex
+}
+
+.ecommerce-header .vlt-site-logo-vonage {
+  display: none
+}
+
+.ecommerce-header .vlt-site-logo-vonage,.ecommerce-header .vlt-site-logo-vonage-simple {
+  flex: 0 1 auto;
+  max-height: 100%;
+  width: auto
+}
+
+.ecommerce-header .vlt-site-logo-vonage-simple {
+  display: flex
+}
+
+.ecommerce-header-order-details {
+  color: #000;
+  display: inline
+}
+
+.ecommerce-header-order-details .d-none {
+  display: block!important
+}
+
+.ecommerce-header-order-details .vlt-sitevng-modal-content-wrapper {
+  margin-top: 0;
+  padding-left: 0;
+  padding-right: 0
+}
+
+.ecommerce-header-order-details .vng-modal-content {
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  background-color: #eeeffc;
+  display: flex;
+  height: 100%;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  padding: 0
+}
+
+.ecommerce-header-order-details .vng-modal-close,.ecommerce-header-order-details .vng-modal-content::before {
+  z-index: 1
+}
+
+.ecommerce-header-order-details .vlt-card-icon-extrusion {
+  display: none
+}
+
+.ecommerce-header-order-details .vlt-card-icon-container {
+  border-radius: 0;
+  -webkit-box-pack: start;
+  -ms-flex-pack: start;
+  justify-content: flex-start;
+  text-align: left;
+  transform: none!important
+}
+
+.vlt-card-icon:hover .ecommerce-header-order-details .vlt-card-icon-container {
+  transform: none!important
+}
+
+.ecommerce-header-order-details .checkout-cart-pricing-details {
+  height: 100%;
+  width: 100%
+}
+
+.ecommerce-header-order-details .checkout-cart-pricing-details-inner {
+  border-top-width: 0;
+  height: 100%
+}
+
+.ecommerce-header-order-details .checkout-cart-pricing-details-pricing-items {
+  display: flex;
+  flex-direction: column;
+  -webkit-box-flex: 1;
+  -ms-flex-positive: 1;
+  flex-grow: 1;
+  margin-top: 1.2rem;
+  padding-top: 1.2rem;
+  min-height: 100vh
+}
+
+.ecommerce-header-order-details .checkout-cart-pricing-details-title {
+  font-size: 2.1rem;
+  padding: 4rem 1.6rem 0
+}
+
+.ecommerce-header-order-details .checkout-cart-pricing-details-btnsharecart,.ecommerce-header-order-details .checkout-cart-pricing-details-fees {
+  margin-bottom: 8rem;
+  padding: 0 1.6rem 7.2rem
+}
+
+.ecommerce-header-order-details .checkout-cart-pricing-details-checkout {
+  bottom: 0;
+  left: 0;
+  margin-top: auto;
+  position: fixed;
+  right: 0;
+  width: 100%
+}
+
+.no-header .ecommerce-header {
+  display: none
+}
+
+@media (min-width: 768px) {
```

Comment:
Consolidate all the media queries into the 3 sizes used instead defining a new media query for each selector.